### PR TITLE
Week three code fixes

### PR DESCRIPTION
**Pull request by JasonGibson274**

### CODE REVIEW

**Comment by JasonGibson274** [comment]
@jgkamat can I can an approval?

File: code/week3/week3.cpp [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+     // ******************** ARRAYS *********************
+     // array docs http://en.cppreference.com/w/cpp/container/array
+
+     // initialization
+     // array<TYPE, SIZE> name = {elements}
+     array<int, 3> int_array = {1, 2, 3};
+     array<string, 3> string_array = {"arrays", "are", "cool"};
+
+     // You can get the number fo elemnts in std::array by using .size()
```
Comment (jgkamat) [inline code/week3/week3.cpp]: `s/fo/of/`

File: code/week3/week3.cpp [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+     cout << "size of the int_arrray is = " << int_array.size() << endl;
+
+     // remember arrays index 0,1,2,...
+     cout << "printing int_array" << endl;
+     for(int i = 0; i < int_array.size(); i++) {
+         // .at(index) returns the element at that index with bounds checking at runtime
+         cout << "string_array at index = " << i << " the value is = " << string_array.at(i) << endl;
+         cout << "string_array at index = " << i << " the value is = " << string_array.at(i) << endl;
```
Comment (jgkamat) [inline code/week3/week3.cpp]: Why is this line being run twice? Maybe copy paste error?

**Review by jgkamat — DISMISSED** [review]
A few nitpicks, but good otherwise

**Review by jgkamat — APPROVED** [review]
LGTM

**Pull request merged**